src/controllers/email.ts

```diff
@@ -30,14 +35,10 @@ export async function sendCode(
   to: string,
   code: string,
 ) {
-  const response = await env.AUTH_TEMPLATES.get(
-    "templates/email/code-v2.liquid",
-  );
-  if (!response) {
-    throw new Error("Code template not found");
-  }
-
-  const templateString = await response.text();
+  // FINAL STEP! these need to be read straight from in here!
+  // A. generate these in E6 strings... just do manually for now right! like M did with tailwind
+  // B. IF WORKS - new PR to generate these. EASY!
+  // C. start asserting and testing... would be good to release but NOT ON FRIDAY lol
```

Comment:
```suggestion
  // FINAL STEP! these need to be read straight from in here!
  // A. generate these in E6 strings... just do manually for now right! like M did with tailwind
  // B. IF WORKS - new PR to generate these. EASY!
  // C. start asserting and testing... would be good to release but NOT ON FRIDAY lol
```

:eyes:  note to self!

I was thinking on the next PR to actually load the email HTML into playwright and snapshot it :crossed_fingers: